components/compliance-service/ingest/pipeline/processor/summary.go

```diff
@@ -63,7 +75,22 @@ func ComplianceShared(in <-chan message.Compliance) <-chan message.Compliance {
 			}
 			msg.Shared.PerProfileSums = perProfileSums
 			msg.Shared.AllProfileSums = totalSum
-			msg.Shared.Status = compliance.ReportComplianceStatus(totalSum)
+
+			if failedProfiles > 0 {
+				// If at least one of the profiles in the report comes in with "failed" status, we ignore the totalSum aggregation which might have all
```

Comment:
thanks for including the comments here and all over, these will be super useful when our future selves try to figure out all this stuff